### PR TITLE
Javadoc for MsbContext

### PR DESCRIPTION
**Pull request by anstr**

### CODE REVIEW

File: core/src/main/java/io/github/tcdl/MsbContext.java [hunk 1]
```diff
@@ -89,8 +112,11 @@ public MsbContextBuilder withDefaultChannelMonitorAgent(boolean withDefaultChann
         }
         
         /**
-         * Create MsbContext and initialize it with Config from reference.conf
-         * @throws MsbException if an error happens during initialization  
+         * Create {@link MsbContext} and initialize it with configuration from reference.conf(property file inside MSB library)
```
Comment (vso-tc) [inline core/src/main/java/io/github/tcdl/MsbContext.java:115]: For actual microservice it's going to read application.conf as well. Doesn't it?

File: core/src/main/java/io/github/tcdl/MsbContext.java [hunk 2]
```diff
@@ -46,22 +49,42 @@ public void shutdown() {
...
+     *
+     * @return factory ({@link ChannelManager}) for creating producers and consumers
+     */
     public ChannelManager getChannelManager() {
         return channelManager;
     }
 
+    /**
+     *
+     * @return object of class {@link Clock} which was crated during constructing MsbContext
```
Comment (vso-tc) [inline core/src/main/java/io/github/tcdl/MsbContext.java]: "that MUST be used to obtain current time.

File: core/src/main/java/io/github/tcdl/MsbContext.java [hunk 3]
```diff
@@ -15,7 +15,10 @@
 import java.time.Clock;
 
 /**
- * {@link  MsbContext} specifies the context of the MSB message processing. Creates singleton beans required for MSB.
+ * Specifies the context of the MSB message processing.
+ * Method {@link MsbContextBuilder#withShutdownHook} is used for specifying shutdown hook during JVM exit.
```
Comment (vso-tc) [inline core/src/main/java/io/github/tcdl/MsbContext.java:19]: Why didn't you put these comments directly at the methods they describe?

Comment (anstr) [inline core/src/main/java/io/github/tcdl/MsbContext.java:19]: At the method more detail descriptions,at the Class level briefly descriptions and link to javadoc for methods.

File: core/src/main/java/io/github/tcdl/MsbContext.java (hunk 2 above)
Comment (anstr) [inline core/src/main/java/io/github/tcdl/MsbContext.java]: Yes,this current time is time when MsbContext object was created.

File: core/src/main/java/io/github/tcdl/MsbContext.java (hunk 1 above)
Comment (anstr) [inline core/src/main/java/io/github/tcdl/MsbContext.java:115]: It's doesn't my line,but I will replace it with "configuration from reference.conf(property file inside MSB library) or application.conf, which will override library properties"

**Pull request merged**